code/game/machinery/bots/medbot.dm

```diff
@@ -451,8 +451,9 @@
 		reagent_id = TOXIN
 
 	var/virus = 0
-	for(var/datum/disease/D in C.viruses)
-		virus = 1
+	for(var/ID in C.virus2)
+		if (ID in virusDB)
```

Comment:
seems medbots only detect if in the virusdb now, make sure it's in the cl

Reply:
previously they only detected old disease, and didn't detect virus2 pathogen at all. I'll put it in the cl, but that mostly means it won't be able to call out stuff like appendicitis, which frankly is no big deal since all it did was inject some spaceacillin that does fuck all to cure appendicitis.